### ISSUE TITLE
MLSD

### ISSUE THREAD

**Issue by morris** [mscdex/node-ftp#107]
Hi!

First of all, thanks for this great package, it backs the new [vinyl-ftp](https://github.com/morris/vinyl-ftp) and turns out to be very, very robust.

However, I needed to implement MLSD for better listings (needed seconds). It's just a small derivation from your LIST code, and I thought you might want to include it. It should be trivial to implement MLST, too. I could also work this into a pull request if you wanted.

https://github.com/morris/vinyl-ftp/blob/master/mlsd.js

**Comment by mscdex** [comment]
I'd be open to a PR adding them as separate methods and not automagically used. Since there is some overlap with the existing dir listing functionality, it would be great to share some of the common code to keep things DRY.

**Comment by morris** [comment]
Definitely, I also use both MLSD and LIST (as a fallback if MLSD is not supported). Hope I get around to filing the PR one of these days...
